packages/volto/src/components/manage/Controlpanels/Controlpanels.jsx

```diff
@@ -130,64 +136,66 @@ function Controlpanels({
     : [];
   const { filterControlPanels } = config.settings;
 
-  const filteredControlPanels = map(
```

Comment:
I don't get the reason for this. Which is the rationale?

Reply:
might not need it after [9a6b25f](https://github.com/plone/volto/pull/3592/commits/9a6b25f08b11f463b7d15b3af3a2151aa6b99407) ?
-cc @wesleybl

Reply:
In https://github.com/plone/volto/commit/3ad97df8467f54d72a1fa1a5c85ce093bef7e3d3, @nileshgulia1 performed a validation to see if there was a controlpanel. I kept that validation. See below:

https://github.com/plone/volto/pull/3592/files#diff-10e8b3997f0087a776219f684dfa7c6440933357dee431c5614d6d4289191c00R139

This prevents an on-screen error if the backend does not return the controlpanels.